functions/index.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
-
-const firebaseFunctions = require('firebase-functions');
-const firebaseAdmin = require('firebase-admin');
-const gcs = require('@google-cloud/storage')();
-const jwt = require('jsonwebtoken');
-const fs = require('fs');
-
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
I think that file was accidentally committed? There is also the `index.ts`. @tinayuangao